<?xml version="1.0"?>
<launch>
  <arg name="robot_ip" default="172.16.0.2" />
  <arg name="load_gripper" default="false" />
  <include file="$(find franka_control)/launch/franka_control.launch">
    <arg name="robot_ip" value="$(arg robot_ip)" />
    <arg name="load_gripper" value="$(arg load_gripper)" />
  </include>

  <group ns="cable_data_collector">
    <rosparam command="load" file="$(find mars_control)/config/move_to.yaml" />
  </group>
  <rosparam command="load" file="$(find mars_control)/config/controllers.yaml" />
  <node name="controller_spawner" pkg="controller_manager" type="spawner" respawn="false" output="screen" args="cable_data_collector" />
</launch>